hnn_core/viz.py

```diff
@@ -1400,7 +1401,7 @@ def _update_section_voltages(self, t_idx):
     def _init_network_plot(self):
         for cell_type in self.net.cell_types:
             gid_range = self.net.gid_ranges[cell_type]
-            for gid_idx, gid in enumerate(gid_range):
+            for gid_idx, _gid in enumerate(gid_range):
```

Comment:
```suggestion
            for gid_idx, _ in enumerate(gid_range):
```

seems like it's not used in the loop?

Reply:
Correct. The flake8 suggestion was to add an underscore to the start of unpacked variables not used in the loop. But we could just use an underscore if that's the style you prefer.